index.html

```diff
@@ -65,10 +65,15 @@ <h1>Step into the<br/>Pluriverse</h1>
 				<p>
 					Read the guide &nbsp;
 					<a href="https://bits.owocki.com/5zuP82qn">[PDF] &nbsp;</a>
-					<a href="https://online.fliphtml5.com/kpnio/fszd/"> [Flipbook]</a>
+					<a href="https://online.fliphtml5.com/wryqh/bdyf/"> [Flipbook]</a>
```

Comment:
drat they still have this really clunky ad on this flipbook too tho.

really wish there was a high fidelity viewer we could use.

Reply:
https://bits.owocki.com/5zuP82qn

also this PDF downloader via cloudapp is a bit clunky.  should we just link directly to the pdf?

Reply:
> drat they still have this really clunky ad on this flipbook too tho.
> 
> really wish there was a high fidelity viewer we could use.

yes, i totally agree with you. For sure i can find another one.

Reply:
> https://bits.owocki.com/5zuP82qn
> 
> also this PDF downloader via cloudapp is a bit clunky. should we just link directly to the pdf?



> https://bits.owocki.com/5zuP82qn
> 
> also this PDF downloader via cloudapp is a bit clunky. should we just link directly to the pdf?

agree on this one as well. i think it would be cleaner with just the pdf

Reply:
the problem is the pdf doesnt show the pages side by side.  idk if theres a way to render the PDF such that it works around this?

<img width="223" alt="Screen Shot 2022-09-01 at 12 07 00 PM" src="https://user-images.githubusercontent.com/513929/187983174-b0b2a2d2-aa67-457b-b63b-70c2d54f65f8.png">

Reply:
Oh, im pretty sure there is a way. I can fix these things tomorrow